components/feature/addons/src/test/java/mozilla/components/feature/addons/ui/AddonsManagerAdapterTest.kt

```diff
@@ -105,4 +111,92 @@ class AddonsManagerAdapterTest {
             }
         }
     }
+
+    @Test
+    fun bindAddon() {
```

Comment:
nit: to keep consistency, I think we should use the backtick(\`bind add-on`\) function names